kategory-core/src/main/kotlin/kategory/typeclasses/Bifoldable.kt

```diff
@@ -0,0 +1,16 @@
+package kategory
+
+interface Bifoldable<in F> : Typeclass {
```

Comment:
Most datatypes that currently support `Bimonad` and `Foldable` are probably gonna be able to implement this typeclass.

Reply:
Which ones ?